active_robot: "robot_0"

available_robots: ["robot_0", "", "", ""]
